GoogleSignIn/Sources/GIDSignInInternalOptions.m

```diff
@@ -25,6 +25,43 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @implementation GIDSignInInternalOptions
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
++ (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
+                       presentingViewController:(nullable UIViewController *)presentingViewController
+                                      loginHint:(nullable NSString *)loginHint
+                                  addScopesFlow:(BOOL)addScopesFlow
+                               verifyCompletion:(nullable GIDVerifyCompletion)completion{
+  GIDSignInInternalOptions *options = [self defaultOptionsWithConfiguration:configuration
+                                                   presentingViewController:presentingViewController
+                                                                  loginHint:loginHint
+                                                              addScopesFlow:addScopesFlow
+                                                     accountDetailsToVerify:@[]
```

Comment:
I think this is a good example of this method providing ambiguous value. I don't understand what we gain by providing a method to supply a set of options to verify no account details?